examples/pirate-ship-app/client/package-lock.json

```diff
@@ -15345,12 +15333,6 @@
               "dev": true,
               "optional": true
             },
-            "ini": {
-              "version": "1.3.5",
-              "bundled": true,
-              "dev": true,
-              "optional": true
-            },
```

Comment:
Why was it deleted instead of updated here?

Reply:
The bot appears to deduplicate. The deps that have it removed use semver and 1.3.8 satisfies the requested range.